src/Commands/AdminCommands/ChatsCommand.php

```diff
@@ -58,6 +58,7 @@ public function execute()
         $results = DB::selectChats(
```

Comment:
This is screaming for bit flags / bitmask:
https://stackoverflow.com/questions/11880360/how-to-implement-a-bitmask-in-php

Having so many boolean parameters is just waaay confusing 😖